### DISCUSSION TITLE
v2.1.7

### DISCUSSION THREAD

**Discussion by RollingTechSupport** [pyroll-project/pyroll-core#213] [Announcements]
We are currently preparing a new minor `pyroll-core` release. 
The release is planned for CW 21.

**Comment by sag-kalibreur** [comment]
Do you already have a changelog of what will change? For example, dealing with units ;-)

**Reply by RollingTechSupport** [reply]
I will start summarizing everything. Integration of units into the core will definitely take some time. 
Currently we are preparing a jupyter notebook that shows you how to handle units in your input and how to simply dump everything into a .pdf.

**Comment by RollingTechSupport** [comment]
## What's Changed
* 196 add longitudinal and latitudinal contact friction by @ChRen95 in https://github.com/pyroll-project/pyroll-core/pull/200
* Add Additional Init Functions to Init the Solution Procedure for Plugins by @axtimhaus in https://github.com/pyroll-project/pyroll-core/pull/202
* added cff by @ChRen95 in https://github.com/pyroll-project/pyroll-core/pull/207
* 195 windows 11 pyroll run gets stuck in an endless loop due to problems with kaleido package by @RichardPfr in https://github.com/pyroll-project/pyroll-core/pull/206


**Full Changelog**: https://github.com/pyroll-project/pyroll-core/compare/v2.1.6...v2.1.7

**Reply by sag-kalibreur** [reply]
Good work, useful additions and bugfixing...
